### PR TITLE
DEX-862 update individual name contexts

### PR DESCRIPTION
**Pull request by Emily-Ke**
Updates the name contexts and labels from defaultName (Name) to FirstName (First name) and from nickname (Nickname(s)) to AdoptionName (Adoption name).

### CODE REVIEW

File: src/models/individual/useIndividualFieldSchemas.js [hunk 1]
```diff
@@ -38,23 +38,23 @@ export default function useIndividualFieldSchemas() {
...
-          labelId: 'NAME',
+          name: 'firstName',
+          labelId: 'FIRST_NAME',
           category: defaultIndividualCategories.general.name,
           requiredForIndividualCreation: true,
           required: true,
           defaultValue: '',
           getValue: (_, individualData) =>
-            deriveIndividualName(individualData, 'defaultName', ''),
+            deriveIndividualName(individualData, 'FirstName', ''),
```
[the remark this answers is not in the document]
Comment (Atticus29) [reply src/models/individual/useIndividualFieldSchemas.js:48]: Maybe this is back end thing or a schema convention I'm not aware of?

File: src/pages/individual/EditIndividualMetadata.jsx [hunk 2]
```diff
@@ -179,42 +179,46 @@ export default function EditIndividualMetadata({
...
-            if (nicknameGuid) {
+            // If there was an adoptionName, update it with whatever the value is now.
+            // If there was not an adoptionName, but there is a value now, add it.
+            // If an adoptionName did not already exist, don't add it.
+            const adoptionNameGuid = deriveNameGuid(
+              metadata,
+              'AdoptionName',
+            );
+            const adoptionNameFieldValue =
+              defaultFieldValues.adoptionName;
```
Comment (Atticus29) [inline src/pages/individual/EditIndividualMetadata.jsx:210]: I didn't quite follow how this is a guaranteed non-null?

File: src/utils/notificationUtils.js [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
   const user1Name =
     notification?.message_values?.user1_name || 'Unnamed User';
   const user2Name =
     notification?.message_values?.user2_name || 'Unnamed User';
-  const individual1Names = notification?.names || []; // TODO flesh out more once this is included in notifications DEX-739
-  const individual1NicknameObject = individual1Names.find(
-    n => n.context === 'nickname',
+
+  // TODO flesh out more once this is included in notifications DEX-739
+  const individual1AdoptionName = deriveIndividualName(
```
Comment (Atticus29) [inline src/utils/notificationUtils.js:11]: thank you for making this use deriveIndividualName

Comment (Emily-Ke) [reply src/utils/notificationUtils.js:11]: The blocker for DEX-739 appears to have been removed, so at some point we should get back to this and not set both names to the same value. 😆

**Review by brmscheiner — APPROVED** [review]

File: src/pages/individual/EditIndividualMetadata.jsx (hunk 2 above)
Comment (Emily-Ke) [reply src/pages/individual/EditIndividualMetadata.jsx:210]: The way I see it, defaultFieldValues is initially set on declaration to the return value from getInitialFormValues, which will be an object. defaultFieldValues is only ever reassigned in the onChange for the edit component. The value that it is set to is at least an empty object. I don't see a way that defaultFieldValues could ever be null, but I may have missed something.

**Pull request merged**